src/Ombi.Notifications/Agents/LegacyMobileNotification.cs

```diff
@@ -241,15 +241,25 @@ protected override async Task Test(NotificationOptions model, MobileNotification
             await Send(playerIds, notification, settings, model);
         }
 
-        private async Task<List<string>> GetAdmins(NotificationType type)
+        private async Task<List<string>> GetAdmins()
```

Comment:
I suggest we just rename this to `GetNotificationRecipients`.

    private async Task<List<string>> GetNotificationRecipients() => await UsersToPlayerIds(await GetAdminAndPowerRecipients());